src/index.ts

```diff
@@ -110,17 +110,18 @@ export function useLocation(): Location {
 
     useEffect(() => {
         /* get last known use position */
-
         getLastLocations(1).then((res) => {
             res[0] && setLocation(res[0])
         }).catch(console.log);
 
-
-
         /* subscribe to the listener */
         const subscription = locationEmitter.addListener('LOCATION_UPDATED', (result: LocationEvent) => {
             if (result.success) {
                 setLocation(result.payload);
+            } else if (result.payload == LocationError.headingFailure) {
+                console.log("Can't get location because of strong interference from nearby magnetic fields")
```

Comment:
I'm not sure if we want to impose logs for the consumer ? Maybe there is a better way, in a promise result, or something else ?

Reply:
I will update it in the next PR. 
But in this hooks we can return, so I still want log an error (but more clearly). 
If the user want, he can call directly the event instead of using the hooks. 
I don't know how do better

Reply:
we can't*